# @author Maik Basso <maik@maikbasso.com.br>

# message header
Header header

# payload
int32 packetLoss
int32 packetReceived
int32 sentMessageCount
int32 numberOfNodes
